python/tchannel/tornado/connection.py

```diff
@@ -75,7 +76,7 @@ class TornadoConnection(object):
     CALL_REQ_TYPES = frozenset([Types.CALL_REQ, Types.CALL_REQ_CONTINUE])
     CALL_RES_TYPES = frozenset([Types.CALL_RES, Types.CALL_RES_CONTINUE])
 
-    def __init__(self, connection):
+    def __init__(self, connection, tchannel=None):
```

Comment:
It will work for current needs. If we consider future extension, we may need pass more things from tchannel. That's why I pass tchannel